GNUmakefile

```diff
@@ -9,7 +9,7 @@ GO_LDFLAGS := "-X github.com/hashicorp/nomad-autoscaler/version.GitCommit=$(GIT_
 .PHONY: tools
 tools: ## Install the tools used to test and build
 	@echo "==> Installing tools..."
-	GO111MODULE=on cd tools && go get -u github.com/golangci/golangci-lint/cmd/golangci-lint@v1.24.0
```

Comment:
I had to remove the update flag since [one of the linters golangci-lint aggregates](https://github.com/dominikh/go-tools/tree/v0.1.0) introduced a breaking change in a minor version release that gets picked up when `-u` is set. This was causing the build to fail.